Neos.Flow/Classes/Mvc/Controller/ActionController.php

```diff
@@ -361,6 +374,7 @@ protected function initializeActionMethodValidators()
             $ignoredArguments = [];
         }
 
+        /* @var $argument Argument */
```

Comment:
Instead we could annotate the `arguments` field with `Argument[]` to expose the type everywhere?!

Reply:
True. Needs to be adjusted in AbstractController